.github/workflows/go_lint.yml

```diff
@@ -13,11 +13,17 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-go@v5
+      # Step 1: Checkout the code from the repository
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+
+      # Step 2: Set up Go environment
+      - name: Set up Go
+        uses: actions/setup-go@41dfa10bad2bb2ae585af6ee5bb4d7d973ad74ed # v5
         with:
           go-version-file: './go.mod'
+
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@818ec4d51a1feacefc42ff1b3ec25d4962690f39 # v6
```

Comment:
This hash is for [v6.4.1](https://github.com/golangci/golangci-lint-action/releases/tag/v6.4.1). Have you considered to use [v6.5.0](https://github.com/golangci/golangci-lint-action/releases/tag/v6.5.0)?